src/components/ebay-carousel/examples/11-items-per-slide-partial-item/template.marko

```diff
@@ -0,0 +1,25 @@
+<style>
+    .demo11-card {
+        color: #cdf4fd;
+        background: #a1208b;
+        font-size: 24px;
+        font-weight: bold;
+        height: 150px;
+        line-height: 150px;
+        text-align: center;
+    }
+</style>
+<ebay-carousel items-per-slide="3.25">
+    <ebay-carousel-item class="demo8-card">Card 1</ebay-carousel-item>
```

Comment:
Although these technically work as-is, they should also be `demo11`. One of these days we'll come up with a better way to handle these classes.

Reply:
Whoops, will fix.